spec/app/ics-101-interchain-swap/README.md

```diff
@@ -16,20 +16,22 @@ This standard document specifies the packet data structure, state machine handli
 
 ### Motivation
 
-ICS-101 Interchain Swaps enables chains their own token pricing mechanism and exchange protocol via IBC transactions.  Each chain can thus play a role in a fully decentralised exchange network.
+ICS-101 Interchain Swaps enables chains their own token pricing mechanism and exchange protocol via IBC transactions. Each chain can thus play a role in a fully decentralised exchange network.
 
 Users might also prefer single asset pools over dual assets pools as it removes the risk of impermanent loss.
```

Comment:
De we need remove this line?, seems `removing risk of impermanent` is not accurate